### ISSUE TITLE
User should be able to see results of a poll

### ISSUE THREAD

**Issue by stanriders** [stanriders/circleforms#5]
- [ ] Sorting by different data points
- [ ] Selectable representation of answers (by poll creator?)
- [ ] Hiding unnecessary data (by poll creator?)
- [ ] Exporting (#7)
- [ ] Different result accessibility (#6)

* [ ] Backend
* [ ] Frontend
* [ ] Design

**Comment by B0und** [comment]
If we want to make results public, then we need a way to make post/id/answers to be available to non-author users
